src/content/docs/setup/analytics/adobe-experience-platform.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Adobe Experience Platform
+description: Learn how to integrate your Adobe Commerce on Edge Delivery Services storefront project to the Adobe Experience Platform.
+---
+
+import OptionsTable from '@components/OptionsTable.astro';
+import Diagram from '@components/Diagram.astro';
+import CodeExample from '@components/CodeExample.astro';
+
+## Overview
+
+Adobe Experience Platform is a suite of services that allows you to collect, manage, and analyze data from your customers. Here's a diagram showcasing the full architecture of Adobe Experience Platform.
+
+<Diagram caption="Adobe Experience Platform architecture.">
+  ![Adobe Experience Platform architecture.](./aep-architecture-1024.png)
+</Diagram>
+
+You can read more about Adobe Experience Platform in the [Adobe Experience Platform documentation](https://experienceleague.adobe.com/docs/experience-platform/landing/home.html).
+
+The documentation on this page will focus on the integration of the Adobe Commerce on Edge Delivery Services storefront project to Adobe Experience Platform.
+
+## Prerequisites
+
+Before you can configure your integration to Adobe Experience Platform you need to have the following:
+
+* IMS Org Id (e.g. `########@AdobeOrg`)
+* Datastream ID (e.g. `########-####-####-####-########`)
+
+{/* You can find your IMS Org Id by... */}
+
+You can find more information about how to create a datastream in the [Adobe Experience Platform documentation](https://experienceleague.adobe.com/docs/experience-platform/datastreams/overview.html).
+
+## Configuration
+
+In order for data to flow from your storefront to the Experience Platform Edge Network you just need to change a few lines of code in your storefront project.
+
+<CodeExample code={`
+window.adobeDataLayer.push({
+		aepContext: {
+				imsOrgId: '########@AdobeOrg',
+				datastreamId: '########-####-####-####-########'
+		},
+		{ eventForwardingContext: { commerce: true, aep: true } },
+	},
+});
+`} lang="js" title="scripts/delayed.js" />
+
+`aepContext` is the context object that contains the IMS Org Id and Datastream ID.
+
+`eventForwardingContext` with `aep: true` tells the Adobe Data Layer to forward the event to the Experience Platform Edge Network and configure it to send to the correct IMS Org and Datastream.
+
+You can see an example of this [here](https://github.com/hlxsites/aem-boilerplate-commerce/commit/eb49d00bcebd21e48cd87c0efced17d2c6ccfd9d).
+
+{/* ## Validation */}
+
+{/* TODO */}
```

Comment:
To validate data ingestion, refer to https://experienceleague.adobe.com/en/docs/platform-learn/getting-started-for-data-architects-and-data-engineers/ingest-batch-data